app/assets/javascripts/foreman_scc_manager/locale/de/foreman_scc_manager.js

```diff
@@ -0,0 +1,592 @@
+ locales['foreman_scc_manager'] = locales['foreman_scc_manager'] || {}; locales['foreman_scc_manager']['de'] = {
+  "domain": "foreman_scc_manager",
+  "locale_data": {
+    "foreman_scc_manager": {
+      "": {
+        "Project-Id-Version": "foreman_scc_manager 1.0.0",
+        "Report-Msgid-Bugs-To": "",
+        "PO-Revision-Date": "2019-10-17 13:28+0000",
+        "Last-Translator": "Ettore Atalan <atalanttore@googlemail.com>, 2023",
+        "Language-Team": "German (https://app.transifex.com/foreman/teams/114/de/)",
+        "MIME-Version": "1.0",
+        "Content-Type": "text/plain; charset=UTF-8",
+        "Content-Transfer-Encoding": "8bit",
+        "Language": "de",
+        "Plural-Forms": "nplurals=2; plural=(n != 1);",
+        "lang": "de",
+        "domain": "foreman_scc_manager",
+        "plural_forms": "nplurals=2; plural=(n != 1);"
+      },
+      " -- Select Architecture --": [
+        ""
+      ],
+      " -- Select Product --": [
+        ""
+      ],
+      " -- Select Version --": [
+        ""
+      ],
+      " has started successfully.": [
+        ""
+      ],
+      "%s/%s": [
+        ""
+      ],
+      "Abstract": [
+        ""
+      ],
+      "Abstract async task": [
+        "Übersicht asynchrone Aufgabe"
+      ],
+      "Action with sub plans": [
+        "Aktion mit Unterplänen"
+      ],
+      "Actions": [
+        "Aktionen"
+      ],
+      "Add SCC account": [
+        "SCC Konto hinzufügen"
+      ],
+      "Add SUSE Customer Center Account": [
+        "SUSE Customer Center Konto hinzufügen"
+      ],
+      "Add more sub products to Product tree": [
+        ""
+      ],
+      "Add product(s)": [
+        ""
+      ],
+      "Additive": [
+        ""
+      ],
+      "Agent action": [
+        "Agentenaktion"
+      ],
+      "Associated GPG key of scc_account": [
+        "Mit SCC Konto assoziierter GPG-Schlüssel"
+      ],
+      "Attach subscriptions": [
+        "Subskriptionen verknüpfen"
+      ],
+      "Auto attach subscriptions": [
+        "Subskriptionen automatisch verknüpfen"
+      ],
+      "Base URL": [
+        "Basis URL"
+      ],
+      "Bulk generate applicability for hosts": [
+        "Anwendbarkeit von Massengenerierung für Hosts"
+      ],
+      "Bulk subscription of scc_products for scc_account": [
+        "Abonnieren mehrere SCC Produkte für ein SCC Konto"
+      ],
+      "Bulk subscription of scc_products with individual repository selection for scc_account.": [
+        ""
+      ],
+      "Collapse products": [
+        ""
+      ],
+      "Collapse/Expand": [
+        ""
+      ],
+      "Combined Profile Update": [
+        "Kombiniertes Profil-Update"
+      ],
+      "Content Only": [
+        ""
+      ],
+      "Copy version units to library": [
+        "Versionseinheiten in Bibliothek kopieren"
+      ],
+      "Create": [
+        "Erstellen"
+      ],
+      "Create Alternate Content Source": [
```

Comment:
E.g. here is from Katello

Reply:
This is because of foreman-tasks, which doesn't filter out actions based on plugin source. @adamruzicka told me there currently is no way to do this. Question is, what's worse? Not having translations, or having a few empty translations?

Reply:
Now regenerated with https://github.com/theforeman/foreman-tasks/pull/735 applied. I think this should address your concern.